tests/input3/src/main/scala/example/Extension.scala

```diff
@@ -0,0 +1,4 @@
+package example
+
+extension (i: Int)
+  def asString: String = i.toString
```

Comment:
You can also add a test for extension group with multiple params using braces.

Reply:
Done